README.md

```diff
@@ -2,38 +2,267 @@
 
 [![codecov](https://codecov.io/gh/Azure/notation-azure-kv/branch/main/graph/badge.svg)](https://codecov.io/gh/Azure/notation-azure-kv)
 
-Azure Provider for the Notary v2 [Notation CLI](https://github.com/notaryproject/notation)
+Azure Provider for the [Notation CLI](https://github.com/notaryproject/notation)
 
-## Getting Started:
-The following summarizes the steps to configure the Azure Key Vault notation plugin, configure gatekeeper, sign and verify a container image to Azure Kubernetes Service
+The notation-azure-kv plugin provides the capability to sign the Notation generated payload by using Azure Key Vault (AKV). The user's certificate and private key should be stored in AKV and the plugin will request signing and getting the leaf certificate from AKV. 
 
-```bash
-# Sign in with Azure CLI.
-# Other authorization methods are also available.
-# See https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorizatio
-az login
+The plugin supports Azure CLI identity and Managed Identity for accessing AKV.
```

Comment:
Are you referring to [AAD workload identities](https://learn.microsoft.com/azure/active-directory/develop/workload-identities-overview) for "Azure CLI identity and Managed Identity"?

Reply:
Azure CLI identity and MI token are the 2 supported itentities for the current version. workload identity is not supported.

Reply:
Can you provide a link? otherwise, customers will be confused.

Reply:
Updated.